sw/device/lib/testing/test_rom/bootstrap.c

```diff
@@ -151,22 +180,25 @@ static int bootstrap_flash(const dif_spi_device_t *spi,
         if (expected_frame_num == 0) {
           // setup default access for data partition
```

Comment:
Nit. Missing full stop here.

Reply:
I'll point out that these style comments are mostly on lines that are not in scope for this PR. If you're asking the contributor if they can help out and clean up the code someone else wrote, that's fine, but you might want to word it that way. ;)